### PR TITLE
CORE-1984: Add chart container component

### PR DESCRIPTION
**Pull request by suchak1**
changes: _rev_

### Proposed Changes
_Please use the Jira Key or NOREF followd by the changes_

- CORE-1984
  - Add chart container component

### Checklist

- [ ] I have added or updated unit tests
- [ ] I have added or updated integration tests (if appropriate)
- [ ] I have added or updated documentation / readme (if appropriate)
- [ ] I have verified that my changes have not introduced new lint errors
- [ ] I have updated the change log

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "virtuoso-design-system",
-  "version": "1.11.2",
+  "version": "1.12.0",
```
Comment (kuguarpwnz) [inline package.json:3]: it updates automatically as well. but idk how it determines which part (minor/patch) should be bumped. @jrschumacher help?

File: stories/vrc-stories/Table.js [hunk 2]
```diff
@@ -113,11 +113,11 @@ storiesOf('Table', module)
     const mode = select('Interaction type', ['none', 'hover', 'clickable'], 'none');
     const isHoverable = mode === 'hover';
     const isClickable = mode === 'clickable';
-    const onClick = isClickable ? d => () => actions.toggle(d.address) : () => undefined;
+    const onClick = isClickable ? (d) => () => actions.toggle(d.address) : () => undefined;
     const isSortable = boolean('Sortable on Address', false);
     const isErrorShown = boolean('Table Error', false);
 
-    const isAllSelected = Object.values(selectedState).every(isTrue => isTrue);
+    const isAllSelected = Object.values(selectedState).every((isTrue) => isTrue);
```
Comment (kuguarpwnz) [inline stories/vrc-stories/Table.js:120]:
```suggestion
    const isAllSelected = Object.values(selectedState).every(Boolean);
```

File: package.json (hunk 1 above)
Comment (suchak1) [reply package.json:3]: ah okay, i'll just undo my manual version bump + changelog and let CI handle it automatically

Comment (jrschumacher) [reply package.json]: @kuguarpwnz @suchak1 we use auto to manage the version. So if you need a minor bump we will apply the `minor` label on the PR. Then the build script will update the changelog and bump.

**Review by NathanAB — APPROVED** [review]

**Comment by virtru-github-public** [comment]
:rocket: PR was released in [`v1.12.0`](https://github.com/virtru/virtuoso-design-system/releases/tag/v1.12.0) :rocket:

**Pull request merged**